source/views/views.js

```diff
@@ -120,6 +120,15 @@ export const allViews: ViewType[] = [
     tint: c.periwinkle,
     gradient: c.lightBlueToBlueDark,
   },
+  {
+    type: 'url',
+    url: 'https://moodle.stolaf.edu/',
```

Comment:
Good that we're using the dynamic URL.

Reply:
Heck yeah. I'm not going to try and remember to update this one every year. Not when I can just do this. 

Side note: how long will it take for a google search for "st. Olaf moodle" to return 17-18 instead of 16-17.

Reply:
Fair point.  FWIW, if we didn't care about keeping the old Moodles accessible, one could feasibly use Google Search Console's [Change of Address](https://support.google.com/webmasters/answer/83106) feature.  (As long as they've verified ownership and stuff)